design/src/main/java/com/ingresse/design/ui/dialog/DSTopSnackbar.kt

```diff
@@ -0,0 +1,58 @@
+package com.ingresse.design.ui.dialog
+
+import android.content.Context
+import android.util.AttributeSet
+import android.widget.RelativeLayout
+import com.ingresse.design.R
+import com.ingresse.design.helper.ResourcesHelper
+import com.ingresse.design.helper.setVisible
+import kotlinx.android.synthetic.main.ds_top_snackbar.view.*
+
+const val VISIBLE_DURATION: Long = 1500
+
+class DSTopSnackbar(context: Context, attrs: AttributeSet): RelativeLayout(context, attrs) {
+    val resHelper = ResourcesHelper(context)
+
+    var positiveIcon = resHelper.getDrawableHelper(R.drawable.ic_check_bamboo)
```

Comment:
Você acessa essas propriedades por fora ou elas podem ser privadas?

Reply:
Deixei público pra se ocorrer de em algum lugar ser diferente quem for usar puder mudar o drawable

Reply:
Mas não seria o ideal fazer por método isso do que alterar diretamente a variável?